python/graphstorm/dataloading/dataloading.py

```diff
@@ -1165,47 +1192,52 @@ def _prepare_negative_sampler(self, num_negative_edges):
         return negative_sampler
 
 class GSgnnLinkPredictionTestDataLoader(GSgnnLinkPredictionDataLoaderBase):
-    """ Link prediction minibatch dataloader for validation and test.
+    """ Mini-batch dataloader for link prediction validation and test.
     In order to efficiently compute positive and negative scores for
-    link prediction tasks, GSgnnLinkPredictionTestDataLoader is designed
-    to only generates edges, i.e., (src, dst) pairs.
+    link prediction tasks, ``GSgnnLinkPredictionTestDataLoader`` is designed
+    to only generates edges, i.e., source and destination node pairs.
 
     The negative edges are sampled uniformly.
 
     Parameters
     -----------
     dataset: GSgnnData
-        The GraphStorm edge dataset
+        The GraphStorm data.
     target_idx : dict of Tensors
-        The target edges for prediction
+        The target edge indexes for link prediction.
     batch_size: int
-        Batch size
+        Mini-batch size.
     num_negative_edges: int
-        The number of negative edges per positive edge
-    fanout: int
-        Evaluation fanout for computing node embedding
+        The number of negative edges per positive edge.
+    fanout: list of int, or dict of list
+         Evaluation fanout for computing node embedding. If it's a dict, it indicates
```

Comment:
```suggestion
         Evaluation fanout for computing node embedding. If it's a dict of list, it indicates
```

Reply:
All fanout arguments are unified into one description:

`Neighbor sampling fanout. If it's a dict of list, it indicates the fanout for each  edge type.`